res/values/strings.xml

```diff
@@ -539,7 +539,7 @@
     <string name="blocked_empty_hint">If you block contacts, they will be shown here.</string>
     <string name="pref_profile_photo_remove_ask">Remove profile image?</string>
     <string name="pref_password_and_account_settings">Password and Account</string>
-    <string name="pref_who_can_see_profile_explain">Your profile image and name will be shown alongside your messages when communicating with other users. Already sent information can not be deleted or removed.</string>
+    <string name="pref_who_can_see_profile_explain">Your profile image, name and signature will be shown alongside your messages when communicating with other users.</string>
```

Comment:
The signature is not "shown alongside your messages", only in the profile and classic MUAs.

Reply:
hm, i regarded the sender profiles as "alongside" :) what to say instead? "Your profile image, name and signature will be sent together with your messages when communicating with other users." ?

Reply:
> "Your profile image, name and signature will be sent together with your messages when communicating with other users." ?

This looks good.

"Shown alongside your messages" was probably referring to the chat view, where your avatar and name are displayed near the message bubble.

Reply:
k, i added a commit.